routes/middleware/authenticate_admin.js

```diff
@@ -0,0 +1,14 @@
+
+// Middleware which authenticates is this user admin or not 
+const authenticate_admin = async (req, res, next) => {
+    try {
+        if (!req.employee) return res.sendStatus(401);
```

Comment:
A 401 response should be saved for when the user is unauthenticated.
This block is used to handle malformed or incomplete request bodies. Therefore we should return a "422 Unprocessable ntity" or "400 Bad Request"